root.go

```diff
@@ -0,0 +1,69 @@
+package main
```

Comment:
Ok if `cobra` has a convention like that then we can go with that no need to change.